docs/en/docs/sql-manual/sql-functions/encrypt-digest-functions/aes.md

```diff
@@ -45,11 +45,11 @@ MySQL> set block_encryption_mode="AES_256_CBC";
 Query OK, 0 rows affected (0.006 sec)
 
 MySQL > select to_base64(AES_ENCRYPT('text','F3229A0B371ED2D9441B830D21A390C3', '0123456789'));
-+----------------------------------------------------------------------------------+
-| to_base64(aes_encrypt('text', 'F3229A0B371ED2D9441B830D21A390C3', '0123456789')) |
-+----------------------------------------------------------------------------------+
-| mvZT1KJw7N0RJf27aipUpg==                                                         |
-+----------------------------------------------------------------------------------+
++-----------------------------------------------------+
+| to_base64(aes_encrypt('text', '***', '0123456789')) |
```

Comment:
what is ****?
what is the changes?

Reply:
https://github.com/apache/doris/pull/7464 （FunctionCallExpr.java Line:266)

Print private key as '***'.